libraries/core_libs/network/rpc/Test.cpp

```diff
@@ -281,7 +281,7 @@ Json::Value Test::get_all_nodes() {
   return res;
 }
 
-Json::Value Test::get_vote_weight(const Json::Value &param1) {
+Json::Value Test::get_coins_per_vote(const Json::Value &param1) {
```

Comment:
I am totally lost why should be this method renamed to get_coins_per_vote ? It still returns vote_weight doesnt it ?

Reply:
It is just rename issue. I accidentally changed it when I was renaming that new config field